editor/editor_file_system.cpp

```diff
@@ -2121,6 +2144,11 @@ void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
 
 	int from = 0;
 	for (int i = 0; i < reimport_files.size(); i++) {
+		if (groups_to_reimport.has(reimport_files[i].path)) {
+			//skip reimporting files that are in groups, as they will be reimported anyways
```

Comment:
Suggested change:

```
// Skip reimporting files that are in groups, as they will be reimported anyways.
```

Reply:
I based this comment style to match the other ones in editor_file_system.cpp, but yes they don't match the comment style guide at https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#comment-style-guide